### PR TITLE
Home page 

### PR DESCRIPTION
**Pull request by swethaogeti**
added videos list ,added  sort video category functionality

### CODE REVIEW

File: src/components/imageSlider/imageSlider.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.carousel {
+  width: 95vw;
+  /* margin-top: 20px; */
+  height: 100%;
+}
+
```
Comment (Rockyraven) [inline src/components/imageSlider/imageSlider.css:6]: remove that unwanted code

File: src/components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  outline: none;
+}
+
+.nav-searchBtn {
+  width: 50px !important;
+  background-color: #fafafa;
+  border-left: 1px lightgray;
+  color: grey;
+  position: absolute;
+  right: 0;
```
Comment (Rockyraven) [inline src/components/navbar/navbar.css:48]: you can also use var color

**Review by Rockyraven — APPROVED** [review]
Do some minar changes otherwise your code is clean and easy to readable

**Comment by abhisheksahani-nova** [comment]
Looks good , clean and readable code , just few basic suggestions and good to merge

1. Use css variables for color across app
2. Remove console.log after testing

**Review by abhisheksahani-nova — APPROVED** [review]

**Pull request merged**